### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by broepke**
Added a thank you :)

### CODE REVIEW

**Comment by broepke** [comment]
Is this ready?

File: README.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ Recently I wrote about how PCA can be used for **image compression** (This repo
 2. **Principal Components Overlayed with the Original Data**: This one is my absolute favorite. You can see the progression of how each principal component brings in slightly more information, and in turn, it becomes hard to distinguish between the different components. This plot is a perfect companion to the Explained Variance Cumulative Plot.
 
 Read more here: https://www.dataknowsall.com/pcavisualized.html
+
+Thank you!
```
Comment (broepke) [inline README.md:18]: That's sweet!

File: README.md [hunk 2]
```diff
@@ -14,3 +14,5 @@ Recently I wrote about how PCA can be used for **image compression** (This repo
 2. **Principal Components Overlayed with the Original Data**: This one is my absolute favorite. You can see the progression of how each principal component brings in slightly more information, and in turn, it becomes hard to distinguish between the different components. This plot is a perfect companion to the Explained Variance Cumulative Plot.
 
 Read more here: https://www.dataknowsall.com/pcavisualized.html
```
Comment (adfrisealach) [inline README.md:16]: thanks for providing a reference

**Review by adfrisealach — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (broepke) [inline README.md:16]: Shouldn't you be wrapping that in a Markdown URL?
```suggestion
Read more here: [https://www.dataknowsall.com/pcavisualized.html](https://www.dataknowsall.com/pcavisualized.html)
```

**Review by broepke — COMMENTED** [review]
All done! LGTM

**Pull request merged**